app/assets/stylesheets/contacts.scss

```diff
@@ -1,5 +1,9 @@
 .page-contacts {
-  .sidebar { padding-bottom: 10px; }
+  .sidebar {
+    max-height: 650px;
```

Comment:
This limits the height of the sidebar on the contact page to 650px, but that doesn't make any sense, because it only makes aspect-management with many aspects more difficult.